lib/common/api/deprecate.js

```diff
@@ -31,6 +31,12 @@ deprecate.warn = (oldName, newName) => {
   return deprecate.log(`'${oldName}' is deprecated. Use '${newName}' instead.`)
 }
 
+deprecate.warnDefault = (propName, oldDefault, newDefault) => {
+  return deprecate.log(`The default value of '${propName}' is changing from \
```

Comment:
There's no need to use `\ ` at the end of line.

Reply:
Without it the log goes across multiple lines